pom.xml

```diff
@@ -431,9 +431,26 @@
                     <artifactId>license-maven-plugin</artifactId>
                     <version>2.0.0</version>
                 </plugin>
+                <plugin>
+                    <artifactId>exec-maven-plugin</artifactId>
+                    <version>1.6.0</version>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <executions>
+                        <execution>
+                            <id>Install git-secrets</id>
+                            <phase>initialize</phase>
+                            <goals>
+                                <goal>exec</goal>
+                            </goals>
+                            <configuration>
+                                <executable>bash</executable>
+                                <commandlineArgs>${project.parent.basedir}/git-hooks/install.sh</commandlineArgs>
```

Comment:
Will this work on Windows? Do we care if not? 

At compliance meeting you said more people were starting to use Windows, although I'm not aware of any on Dockstore team. And if they do, they could presumably use WSL. But, checking to be safe.

Reply:
> Windows users as in CLI users?
> Developers but off the Dockstore team?

We had just had a meeting where it was discussed that some engineers at UCSC might be starting to use Windows machines, so it was fresh on my mind when I was reviewing this PR. But no Windows machines on Seabright team, so I think we should probably just ignore this thread that I started here. :)